### ISSUE TITLE
BigDecimal support

### ISSUE THREAD

**Issue by AndyNortrup** [realm/realm-java#461]
It would be nice to have native big decimal support in the Java library as it is the recommended class for storing and working with currency values in Java.

**Comment by nico-dangerous** [comment]
Hi,
Any progress here ?

**Comment by cmelchior** [comment]
Not really, sorry. The solution right now would be to store it as a String and do the conversion in get/set methods.

**Comment by nico-dangerous** [comment]
Thanks,
I will try that.

**Comment by Zhuinden** [comment]
This is one of those things that type adapters would handle quite nicely.

**Comment by kneth** [comment]
I think we need at decimal type in Realm Core. @rrrlasse How is it to implement?
